test/test_helper.rb

```diff
@@ -16,6 +16,8 @@
 require 'mocha/minitest'
 
 FactoryBot::SyntaxRunner.class_eval do
+  def self.file_fixture_path; end
```

Comment:
I would prefer to wait until factory_bot_rails is updated to do this itself.

Reply:
I looked into this a bit further, it isn't actually an issue with factory_bot, but by `include ActionDispatch::TestProcess`, which we need to call `file_fixture_upload` inside a `FactoryBot.define`. 
The other option would be to rework the factory for images so we don't depend on `file_fixture_upload`

Reply:
In that case, I would actually implement this correctly, so make this return `Rails.root.join('test/files')`.

Reply:
I don't think using `file_fixture_path` is necessarily the proper way to do this. Rails check if a file exists and uses `file_fixtures_path` (and previously `fixtures_path`) if that isn't the case. So using `file_fixtures_upload` with the full path isn't wrong. Again, see commit for this change: https://github.com/rails/rails/commit/beb7fba632b19a6b3c021b616611048bde50f5a4

Moreover, it seems to me that file_fixture_path is not something that you are supposed to modify, but should always be 'test/fixtures/files'. If we want to switch test files to that location, I would do so in a separate PR, so we can always use `file_fixture` instead of the `Rails.root.join('test/files/...')`

Reply:
In any case, this solution doesn't seem correct to me. Either the method is added here and it makes sense to do so (and in that case it should be implemented sensibly as well), or the method should not be added here and the way images are added in the factories right now is wrong.

Reply:
I've removed this method and used Rack::Test to upload files.
I tried to make this work with `file_fixture_upload` and a correct `file_fixture_path`, but this proved to be very complicated to make everything work. Rack::Test seems like an easier solution